react/features/feedback/components/FeedbackDialog.web.js

```diff
@@ -256,11 +256,29 @@ class FeedbackDialog extends Component<Props, State> {
                             shouldFitContainer = { true }
                             value = { message } />
                     </div>
+                    {interfaceConfig.SHOW_REPORT_A_PROBLEM && (
+                        <a
+                            className = 'report-a-problem'
+                            onClick = { this._onReportProblemClick }>
+                            { t('feedback.reportProblem') }
+                        </a>
+                    )}
                 </div>
             </Dialog>
         );
     }
 
+    /**
+     * Dispatches an action notifying that the 'Report a problem' button was pressed.
+     *
+     * @returns {void}
+     */
+    _onReportProblemClick = () => {
+        if (typeof APP !== 'undefined') {
+            APP.API.notifyReportAProblemPressed(window.location.href);
```

Comment:
IMHO we don't need to report the URL but maybe I'm missing something.

Just FYI: `window.location.href` may not include the hash and query params passed initially when the iframe is created. We are stripping them from URL at some point.